PyLint fixes for salt/utils/vmware.py

samilaine opened the pull request:
### What does this PR do?

> Your code has been rated at 9.67/10 (previous run: 8.96/10, +0.70)

PyLint corrections to facilitate further work (at some undefined time in the future).

### What issues does this PR fix or reference?

### Tests written?

No

### Commits signed with GPG?

No

samilaine committed:
Your code has been rated at 9.67/10 (previous run: 8.96/10, +0.70)

rallytime commented on salt/utils/vmware.py:
```diff
@@ -432,9 +431,9 @@ def get_new_service_instance_stub(service_instance, path, ns=None,
-    #For python 2.7.9 and later, the defaul SSL conext has more strict
-    #connection handshaking rule. We may need turn of the hostname checking
-    #and client side cert verification
+    # For python 2.7.9 and later, the defaul SSL conext has more strict
```
`defaul` --> `default`

I know that wasn't really what you're changing in here, but since you've edited the line anyway, might as well fix that up :)

samilaine replied:
I find it hard to disagree on that one. While beautifying, one might as well *read* it too ...

rallytime reviewed (approved):
Just one small comment, otherwise 👍

samilaine committed:
In the defence of grammar.

rallytime reviewed (approved):
🎉

rallytime commented:
Hey @samilaine - This change is causing some of the vmware tests to fail:

- unit.utils.vmware.test_license.AssignLicenseTestCase.test_update_assigned_licenses_call_with_entity
- unit.utils.vmware.test_license.AssignLicenseTestCase.test_update_assigned_licenses_vcenter
- unit.utils.vmware.test_host.GetHostsTestCase.test_get_si_datacenter_name_and_cluster_name

https://jenkins.saltstack.com/job/PR/job/salt-pr-linode-ubuntu14-n/23816/

Can you take a look?

samilaine commented:
re-run all

samilaine committed:
Corrections for failed tests:

   unit.utils.vmware.test_license.AssignLicenseTestCase.test_update_assigned_licenses_call_with_entity
   unit.utils.vmware.test_license.AssignLicenseTestCase.test_update_assigned_licenses_vcenter
   unit.utils.vmware.test_host.GetHostsTestCase.test_get_si_datacenter_name_and_cluster_name

The first two failed due to what appears like a brainfart in the tests
themselves. The last one was corrected along the same lines of thought,
though with less confidence in the chosen solution.
